autobot/github.py

```diff
@@ -17,232 +17,217 @@
 from autobot.config_loader import Config
 
 
-class GitHubAPI:
-    """Perform requests for the target repositories."""
+class GHWrapper:
+    """GitHub object wrapper."""
 
-    def __init__(self, owner, gh_token):
-        """Github client initialization."""
-        self.OWNER = owner
-        self.GH_CLIENT = login(token=gh_token)
+    def __getattr__(self, attr):
+        """Override __getattr__ method."""
+        return getattr(self.__dict__, attr, getattr(self.__dict__["_gh_obj"], attr))
```

Comment:
```suggestion
        return getattr(self, attr, getattr(self._gh_obj, attr))
```

I'm suspecting you had to do this because you were getting recursion errors. I would say It's better to find out why the recursion errors happened in the first place and address the issue there.